packages/cel/src/func.ts

```diff
@@ -95,11 +107,27 @@ export function celOverload<
   const P extends readonly CelType[],
   const R extends CelType,
 >(
+  id: string,
```

Comment:
@jafaircl why was it necessary to add the ID back? We had it previously, but didn't find it useful.

Reply:
It’s used when resolving overloads to determine if the overload should be disabled (i.e. cross type numeric comparisons or user defined disabled overloads). See the resolveOverload method on line 522 in checker.ts. It’s also needed for reference info which is part of the cel spec (https://github.com/google/cel-spec/blob/121e265b0c5e1d4c7d5c140b33d6048fec754c77/proto/cel/expr/checked.proto#L387)